extensions/amp-analytics/0.1/variables.js

```diff
@@ -220,7 +220,7 @@ export class VariableService {
         // Expand string values further.
         p = this.expandTemplate(raw,
             new ExpansionOptions(options.vars, options.iterations - 1,
-                options.noEncode));
+                true /* noEncode */));
```

Comment:
Does this mean JSON provided values aren't encoded the first time anymore? That could lead to XSS issues.